### ISSUE TITLE
Resize handler not removed when fit property is true

### ISSUE THREAD

**Issue by ryanthomasnz** [plotly/react-plotly.js#24]
When a Plotly-React plot is removed from page (props.fit is true) the following error occurs when resizing the window:
```
Uncaught TypeError: Cannot read property 'parentElement' of null
    at PlotlyComponent.getParentSize (factory.js:251)
    at PlotlyComponent.getSize (factory.js:264)
    at resizeHandler (factory.js:195)
getParentSize @ factory.js:251
getSize @ factory.js:264
resizeHandler @ factory.js:195
```

**Comment by bpostlethwaite** [comment]
Thanks for the report @ryanthomasnz . We'll get back to you sometime in early-mid January.

**Comment by nicolaskruchten** [comment]
This is fixed in the latest release, 1.3.0... Thanks for the bug report!
